contracts/hardhat.config.js

```diff
@@ -38,7 +38,12 @@ module.exports = {
     rinkeby: {
       url: process.env.RINKEBY_URL || "",
       accounts:
-        process.env.PRIVATE_KEY !== undefined ? [process.env.PRIVATE_KEY] : [],
+        process.env.RINKEBY_PRIVATE_KEY !== undefined ? [process.env.RINKEBY_PRIVATE_KEY] : [],
+    },
+    polygon_mumbai: {
+      url: process.env.POLYGON_MUMBAI_URL || "",
+      accounts:
+        process.env.POLYGON_MUMBAI_PRIVATE_KEY !== undefined ? [process.env.POLYGON_MUMBAI_PRIVATE_KEY] : [],
```

Comment:
Codacy has a fix for the issue: [Replace `·?·[process.env.POLYGON_MUMBAI_PRIVATE_KEY]` with `⏎··········?·[process.env.POLYGON_MUMBAI_PRIVATE_KEY]⏎·········`](https://app.codacy.com/gh/corkspork/micro-lottery/pullRequest?prid=8883826)
```suggestion
        process.env.POLYGON_MUMBAI_PRIVATE_KEY !== undefined
```